tests/expectations/compiler/finalize/get_or_incorrect_type_fail.out

```diff
@@ -1,9 +1,4 @@
 DCE_ENABLED:
-Error [ETYC0372031]: A mapping's value cannot be a record
```

Comment:
So these errors are now gone but I think that's okay. I see that this particular error is checked in other tests such as `finalize/mapping_fail.leo`